modules/images/webp-uploads/load.php

```diff
@@ -2,7 +2,6 @@
 /**
```

Comment:
Can you also move all `webp-uploads`-related test files accordingly?

Reply:
👍🏼